### PR TITLE
Feature/about command

### PR DESCRIPTION
**Pull request by SeveralCircles**
Added a command which can be used as a template for other commands. Otherwise it displays information about the plugin. Also added a plugin.yml.

### CODE REVIEW

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/Command_about.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

File: src/com/severalcircles/colors/commands/Command_about.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package com.severalcircles.colors.commands;
+
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class Command_about implements CommandExecutor {
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/Command_about.java]: Codacy Issue found: [The class name 'Command_about' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646)

File: src/com/severalcircles/colors/commands/Command_about.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    @Override
```
Comment (SeveralCircles) [inline src/com/severalcircles/colors/commands/Command_about.java]: Codacy Issue found: [Annotation 'Override' should be on the same line with its target.](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646)

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandAbout.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandAbout.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandAbout.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandAbout.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

**Comment by SeveralCircles** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/com/severalcircles/colors/commands/CommandAbout.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/aidanveney/colors/pullRequest?prid=5519646&bid=18084466)

**Review by BeegBrainMo — APPROVED** [review]
Yes

**Pull request merged**